storage/innobase/row/row0mysql.cc

```diff
@@ -2193,51 +2295,23 @@ row_update_cascade_for_mysql(
                 return(DB_FOREIGN_EXCEED_MAX_CASCADE);
         }
 
-	trx_t* trx = thr_get_trx(thr);
+	const trx_t* trx = thr_get_trx(thr);
 
-	bool vers_set_fields = node->table->versioned()
-				  && (node->is_delete == PLAIN_DELETE
-				      || node->update->affects_versioned());
+	bool vers_set_fields = table->versioned()
+				&& (node->is_delete == PLAIN_DELETE
+				    || node->update->affects_versioned());
```

Comment:
The indentation is still too deep. The second line should be indented by one extra TAB.
The relative indentation of the 3rd line to the 2nd line looks correct.